spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-config-sample/README.adoc

```diff
@@ -0,0 +1,78 @@
+= Spring Cloud GCP Pub/Sub Configuration Management Code Sample
```

Comment:
I think this is one of those samples that can greatly benefit from a simple architectural diagram. WDYT?
Maybe we can also re-use it in the refdoc.